### PR TITLE
Update italian language

### PR DESCRIPTION
**Pull request by bovirus**
@auouymous 

Please merge it. Thanks.

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -1886,34 +1888,35 @@ msgid ""
...
 "Facendo clic sul blocco spunte e lasciandolo vuoto disabiliterà l'auto-"
-"scaricamento per tutti gli episodi in questo canale.  I modelli "
-"corrispondono al testo parziale nel titolo dell'episodio, e uno schema vuoto "
-"corrisponde a qualunque titolo. Il modello eccetto sblocca gli episodi "
-"bloccati (per bloccare tutti e poi sbloccarne alcuni)."
+"scaricamento per tutti gli episodi in questo canale. \n"
+"I modelli corrispondono al testo parziale nel titolo dell'episodio, e uno "
+"schema vuoto corrisponde a qualunque titolo. \n"
+"Il modello eccetto sblocca gli episodi bloccati (per bloccare tutti e poi "
+"sbloccarne alcuni)."
```
Comment (auouymous) [inline po/it.po:1895]: Does the last sentence indicate it blocks or unblocks filtered episodes, or that it locks or unlocks them like the archive menu item?

**Review by auouymous — CHANGES_REQUESTED** [review]

File: po/it.po [hunk 2]
```diff
@@ -503,15 +506,15 @@ msgstr "A_pplica"
...
+msgstr "Caricaemnto iscrizioni"
 
 #: src/gpodder/gtkui/main.py:661
 msgid "Your subscriptions are being uploaded to the server."
 msgstr "Caricamento delle iscrizioni sul server in corso."
 
 #: src/gpodder/gtkui/main.py:666
 msgid "List uploaded successfully."
-msgstr "Elenco caricato con successo."
+msgstr "Caricamento elenco completato correttamente."
```
[the remark this answers is not in the document]
Comment (bovirus) [reply po/it.po:517]: For me is enough
"Caricamento elenco completato."

File: po/it.po (hunk 1 above)
Comment (bovirus) [reply po/it.po:1895]: Please take care that there is a pending issues about italian language update

https://github.com/gpodder/gpodder/pull/1537

Comment (auouymous) [reply po/it.po:1895]: What?

Comment (bovirus) [reply po/it.po:1895]: We cannot discuss about an old version of translation until author not include latest pending integration of italian tarnslation.

Comment (auouymous) [reply po/it.po:1895]: The translation can not be merged until these review questions have been answered.

Comment (bovirus) [reply po/it.po:1895]: @auouymous 

I forgot that you are the author (I beleive .... ;-)

Itunblock the episodes basing on exception model.

I will do a small change in

"Il modello eccezione sblocca gli episodi bloccati (per bloccarli tutti e poi "
"sbloccarne soilo alcuni)."

Thanks.

Note:; if you can please avoid in .poo file in teh string the automatic wordwrap at 80 chars:
It cannot help on long strings.
What's teh comamnd to sue to format the string in .po file?

Comment (auouymous) [reply po/it.po:1895]: > I forgot that you are the author

You should answer any review questions, even if the user isn't associated with the project. Members can close irrelevant questions, but valid issues must be resolved, no matter who asks them.

> please avoid in .poo file in teh string the automatic wordwrap at 80 chars:
> What's teh comamnd to sue to format the string in .po file?

@elelay Do you know the answer to that?

Comment (elelay) [reply po/it.po:1895]: commands are run from the makefile. `msgmerge` has a `--no-word-wrap` option but I don't see why we should use it. @bovirus what's your problem again?

Comment (bovirus) [reply po/it.po:1895]: @elelay

The problem is about long string.
in your .po files seems wordwrap automatically at 80 chars.
Then long string are wordwrapped and when you translate it if you sue a simple text editor (and not POEdit) is mroe complex to revise and edit string.
For transltor can help to have long string without wordwrao at 80 chars.

Comment (bovirus) [reply po/it.po:1895]: @elelay elelay

Example

** Now

msgid ""
"Clicking the block checkbox and leaving it empty will disable auto-download "
"for all episodes in this channel.  The patterns match partial text in "
"episode title, and an empty pattern matches any title.  The except pattern "
"unblocks blocked episodes (to block all then unblock some)."
msgstr ""
"Facendo clic sul blocco spunte e lasciandolo vuoto disabiliterà l'auto-"
"scaricamento per tutti gli episodi in questo canale. \n"
"I modelli corrispondono al testo parziale nel titolo dell'episodio, e uno "
"schema vuoto corrisponde a qualunque titolo. \n"
"Il modello eccetto sblocca gli episodi bloccati (per bloccare tutti e poi "
"sbloccarne alcuni)."

** My idea (without word-wrap)

msgid "Clicking the block checkbox and leaving it empty will disable auto-download for all episodes in this channel. \nThe patterns match partial text in episode title, and an empty pattern matches any title.\nThe except pattern unblocks blocked episodes (to block all then unblock some).
msgstr "Facendo clic sul blocco spunte e lasciandolo vuoto disabiliterà l'auto-scaricamento per tutti gli episodi in questo canale.\nI modelli corrispondono al testo parziale nel titolo dell'episodio, e uno schema vuoto corrisponde a qualunque titolo. \nIl modello eccetto sblocca gli episodi bloccati (per bloccare tutti e poi sbloccarne alcuni)."

I hope that is now more clear.

**Comment by auouymous** [comment]
I need to merge some other PRs to get a release out. Any chance you could finish the above reviews so you won't need to rebase this PR?

File: po/it.po [hunk 3]
```diff
@@ -503,15 +506,15 @@ msgstr "A_pplica"
 
 #: src/gpodder/gtkui/main.py:660
 msgid "Uploading subscriptions"
-msgstr "Caricamento delle iscrizioni"
+msgstr "Caricaemnto iscrizioni"
```
[the remark this answers is not in the document]
Comment (bovirus) [reply po/it.po:509]: I don't know.
The message is refering generally to subscription (without additional info about whiuck type of subwscription).

File: po/it.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (bovirus) [reply po/it.po:517]: Yes. Should be laoding of subscription.

**Comment by bovirus** [comment]
@auouymous

For me the iatlian language file is OK.
I answered to all your requests.

**Comment by auouymous** [comment]
Thank you for the translation.

**Pull request merged**